example_school_library_decorator/person.rb

```diff
@@ -1,27 +1,28 @@
-require './corrector.rb'
+require './corrector'
```

Comment:
- [ ] A method for adding rentals should be present. I.E A person should have many rentals as from the requirements:
![image](https://user-images.githubusercontent.com/68167810/132858606-15bdc6bc-d7ce-4b6f-ad59-f1a8b68d65d3.png)
- [ ] The attr_accessor for rentals is also missing. Kindly update it. Also, remember to initialize it in the constructor.